pyro/distributions/gamma.py

```diff
@@ -94,3 +95,34 @@ def analytic_var(self):
         Ref: :py:meth:`pyro.distributions.distribution.Distribution.analytic_var`
         """
         return self.alpha / torch.pow(self.beta, 2.0)
+
+
+class TorchGamma(TorchDistribution):
+    """
+    Compatibility wrapper around
+    `torch.distributions.Gamma <http://pytorch.org/docs/master/_modules/torch/distributions.html#Gamma>`_
+    """
+    reparameterized = True
```

Comment:
@martinjankowiak you should be able to test reparameterized `Gamma` on this branch